crypt_keeper.gemspec

```
@@ -18,8 +18,8 @@ Gem::Specification.new do |gem|
 
   gem.post_install_message = "WARNING: CryptKeeper 2.0 contains breaking changes and may require you to reencrypt your data! Please view the README at https://github.com/jmazzi/crypt_keeper for more information."
 
-  gem.add_runtime_dependency 'activerecord',  '>= 4.2', '< 6.1'
-  gem.add_runtime_dependency 'activesupport', '>= 4.2', '< 6.1'
+  gem.add_runtime_dependency 'activerecord',  '>= 4.2', '<= 6.1'
+  gem.add_runtime_dependency 'activesupport', '>= 4.2', '<= 6.1'
```

```suggestion
  gem.add_runtime_dependency 'activerecord',  '>= 4.2', '< 6.2'
  gem.add_runtime_dependency 'activesupport', '>= 4.2', '< 6.2'
```

Now that Rails 6.1.1 just came out I got the error:

```
$ bundle update rails crypt_keeper
Fetching https://github.com/nikz/crypt_keeper.git
Fetching gem metadata from https://rubygems.org/.........
Fetching gem metadata from https://rubygems.org/.
Resolving dependencies...
Bundler could not find compatible versions for gem "activesupport":
  In Gemfile:
    crypt_keeper was resolved to 2.1.0, which depends on
      activesupport (>= 4.2, <= 6.1)

    railties (~> 6.1.1) was resolved to 6.1.1, which depends on
      activesupport (= 6.1.1)
```

I believe the above change will support any 6.1.x version of rails. Could you please update?